Attempting gateway doc import from previous repo

jhadobe opened the pull request:
Importing gateway `.md` files from: https://github.com/AdobeDocs/commerce-extensibility

keharper commented on src/pages/guides/gateway/create-tenant.md:
````diff
@@ -0,0 +1,142 @@
+
+The following example adds both an Adobe Commerce instance (with Live Search enabled) and an Adobe Experience Manager instance to the mesh. The GraphQL endpoints for Commerce and Live Search are different, therefore you must configure them separately.
+
+    ```json
````
The code sample needs to be left justified. Also, the code gates need to be closed after line 61.

jhadobe replied:
it looks like this one needs to stay indented or it break numbering.

keharper commented on src/pages/guides/gateway/create-tenant.md:
````diff
@@ -0,0 +1,142 @@
+
+After creating a tenant, you should be able to access the GraphQL endpoint by entering the following URL in any GraphQL browser:
+
+```https://graph.adobe.io/api/<chosen_tenant_name>/graphql?api_key=<your_apiKey>```
````
```suggestion
`https://graph.adobe.io/api/<chosen_tenant_name>/graphql?api_key=<your_apiKey>`
```

keharper commented on src/pages/guides/gateway/create-tenant.md:
````diff
@@ -0,0 +1,142 @@
+
+When updating a tenant, do not include a tenantId in your mesh `JSON` file. Compare the following example to the previous [Creating a tenant](#creating_a_tenant) example on this page and note the absence of the `tenantId`.
+
+    ```json
````
Left justify the code sample

keharper reviewed (commented):
I'm not reading the text, just looking for potential problems.

jhadobe commented:
This comment is to contextualize my latest commit: https://github.com/AdobeDocs/graphql-mesh-gateway/pull/7/commits/dd50db74edccbc886af52c6dff97f48e772db045

I added all files to the side nave. I used the header navigation because it looks the cleanest for now. 

I added a .env file, which is necessary for the `Edit in Github` and `Log an Issue` buttons to work. 

Since we can add html elements, we could use <details> to collapse a section. I used one in the create-tenant.md page to keep the code from obscuring the process. I also added a <p> block to fix some spacing. 

I went through and resolved the issues and display errors that I could figure out, here are some remaining issues:

- The first paragraph of every page is in a slightly larger font with extra space after the paragraph. 
- Adobe Graph is listed twice in the breadcrumbs. Modifying the top "Pages:" entry in the Gatsby config seems to change both of them.
- Lists and indentations seem to have general issues. Having two lists back to back causes them to appear as double spaced. Lists appear very close, vertically, to the sentence that precedes them. 
- Separate lines that were defining key value pairs somehow wrapped to a single line. I'm guessing it has to do with the way the JSX parser looks at `code` blocks. 
- Using a note in a list breaks the numbering. Also, indenting the note usually prevents the note from displaying properly. Following what I could find in PWA, I replaced notes in lists with a simple "NOTE: ..." option. 
- Not sure where to customize the Console and "Was this helpful" Yes/No buttons

keharper commented on src/pages/guides/gateway/command-reference.md:
````diff
@@ -0,0 +1,117 @@
+
+#### Response
+
+```bash
````
```suggestion
```terminal
```

keharper commented on src/pages/guides/gateway/command-reference.md:
````diff
@@ -0,0 +1,117 @@
+
+#### Response
+
+```bash
````
```suggestion
```terminal
```

keharper commented on src/pages/guides/gateway/create-tenant.md:
```diff
@@ -0,0 +1,147 @@
+
+    **NOTE:** The following example adds both an Adobe Commerce instance (with Live Search enabled) and an Adobe Experience Manager instance to the mesh. The GraphQL endpoints for Commerce and Live Search are different, therefore you must configure them separately.
+
+   <p></p>
```
This example is not that long. Remove the collapsing markup.

keharper commented on src/pages/guides/gateway/command-reference.md:
````diff
@@ -0,0 +1,117 @@
+
+#### Response
+
+```bash
````
```suggestion
```terminal
```

keharper commented on src/pages/guides/gateway/source-handlers.md:
```diff
@@ -0,0 +1,173 @@
+---
+title: Add source handlers
```
Add a description

keharper commented on src/pages/guides/gateway/transforms.md:
```diff
@@ -0,0 +1,254 @@
+---
+title: Transforms
```
Add a description

keharper reviewed (commented):
A few comments for now. I didn't read the content, but I left a comment where I saw something amiss.
